### PR TITLE
`longflag` is required, `name` is optional in SEM XML

### PR DESCRIPTION
**Pull request by dzenanz**
This should spare someone else from the same issue I encountered [here](https://github.com/Slicer/SlicerExecutionModel/issues/140) by changing the input and output image from positional to flag argument.

### CODE REVIEW

File: Utilities/Templates/Modules/CLI/TemplateKey.xml [hunk 1]
```diff
@@ -12,22 +12,22 @@
     <label>IO</label>
     <description><![CDATA[Input/output parameters]]></description>
     <double>
-      <name>sigma</name>
+      <name>sigma</name><!-- This is the variable name in C++ source code. If unspecified, it is taken from longflag. -->
```
Comment (lassoan) [inline Utilities/Templates/Modules/CLI/TemplateKey.xml:15]:
```suggestion
      <name>sigma</name><!-- This is the variable name in C++ source code. If unspecified, C++ variable name is taken from longflag. -->
```

**Review by lassoan — COMMENTED** [review]
Thank you, clarification in the comment is useful. Why did you change name to longflag for the input and output volumes?

**Comment by dzenanz** [comment]
This linked issue explains the problem:
https://github.com/Slicer/SlicerExecutionModel/issues/140

Making these changes in the template file should reduce likelihood of this problem reoccurring.

**Comment by dzenanz** [comment]
ping

**Review by lassoan — APPROVED** [review]
Thank you for your contribution. I've added further details about what elements are required and also updated SlicerExecutionModel to include a check for missing longflag attribute at compile time.

**Pull request merged**